testcases/kernel/security/selinux/selinux.c

```diff
@@ -0,0 +1,55 @@
+#include <errno.h>
+#include <fcntl.h>
+
+#include "test.h"
+#include <selinux/selinux.h>
+
+
+char *TCID = "selinux";
+int TST_TOTAL = 1;
+
+static void cleanup(void);
+static void setup(void);
+
+static int old_enforcing;
+
+static void setup(void)
+{
+	tst_require_root();
+	tst_tmpdir();
+
+	if (!is_selinux_enabled())
+		tst_brkm(TBROK, cleanup, "selinux not enabled\n");
+
```

Comment:
This is ```TCONF``` and not ```TBROK```. The ```TBROK``` means that some of the initialization steps has failed while ```TCONF``` means that test is not applicable.